source/databricks/settlement_report/settlement_report_job/utils.py

```diff
@@ -284,3 +285,32 @@ def merge_files(
         dbutils.fs.mv("file:" + str(tmp_dst), str(dst))
 
     return list(set([str(_file.dst) for _file in new_files]))
+
+
+def should_include_ephemeral_grid_area(
+    calculation_id_by_grid_area: dict[str, UUID] | None,
+    grid_area_codes: list[str] | None,
+    split_report_by_grid_area: bool,
+) -> bool:
+    return (
+        _check_if_only_one_grid_area_is_selected(
+            calculation_id_by_grid_area, grid_area_codes
+        )
+        or split_report_by_grid_area
+    )
+
+
+def _check_if_only_one_grid_area_is_selected(
```

Comment:
What this function does is not intuitive based on it's name. Does it e.g. throw if more grid area are selected?
```suggestion
def _is_exactly_one_grid_area_selected(
```

Reply:
I have renamed it to your suggestion, Bjarke